eng/pipelines/templates/Build_and_UnitTest.yml

```diff
@@ -401,6 +384,40 @@ steps:
     dropMetadataContainerName: 'DropMetadata-ProfilingInputs'
   condition: "and(succeeded(), eq(variables['BuildRTM'], 'false'), eq(variables['IsOfficialBuild'], 'true'))"
 
+- task: MicroBuildBuildVSBootstrapper@2
+  displayName: 'Build a Visual Studio bootstrapper for tests'
+  inputs:
+    channelName: "$(VsTargetChannelForTests)"
+    vsMajorVersion: "$(VsTargetMajorVersion)"
+    manifests: '$(Build.Repository.LocalPath)\artifacts\VS15\Microsoft.VisualStudio.NuGet.Core.vsman'
+    outputFolder: '$(Build.Repository.LocalPath)\artifacts\VS15'
+  condition: "and(succeeded(), eq(variables['BuildRTM'], 'false'))"
+
+- task: PublishBuildArtifacts@1
+  displayName: 'Publish BootstrapperInfo.json as a build artifact'
+  inputs:
+    PathtoPublish: $(Build.StagingDirectory)\MicroBuild\Output
+    ArtifactName: MicroBuildOutputsForTests
+    ArtifactType: Container
+  condition: "and(succeeded(), eq(variables['BuildRTM'], 'false'))"
+
+- task: PowerShell@1
+  displayName: "Set Bootstrapper URL variable for tests"
+  name: "vsbootstrapper"
+  inputs:
+    scriptType: "inlineScript"
+    inlineScript: |
+      try {
+        $json = Get-Content "${env:Build_StagingDirectory}\MicroBuild\Output\Bootstrapperinfo.json" | ConvertFrom-Json
+        $bootstrapperUrl = $json[1].bootstrapperUrl;
```

Comment:
Do we have any guarantees that the second bootstrapper will always be index `[1]`?  Perhaps Nikolche's suggestion of building the bootstrapper in a second directory will remove the risk.

Reply:
> Do we have any guarantees that the second bootstrapper will always be index [1]? Perhaps Nikolche's suggestion of building the bootstrapper in a second directory will remove the risk.

Thank you @zivkan and @nkolev92 for the review comments because they pushed me to get the cleaner approach working. When I created this PR, I know the proposed solution is hacky to unblock official builds. That's why I mentioned `There are few improvements/clean-ups that can be made to this PR, but I would like to merge it as soon as possible to unblock official CI builds and do the clean-ups in the follow-up PR.` in the PR description. Please review the latest commit and let me know your feedback. 

 [OptProf pipeline](https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=6905358&view=results) and [official build](https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=6904450&view=results) succeeded with the latest changes.